### ISSUE TITLE
Underline links in visual mode

### ISSUE THREAD

**Issue by 0nko** [wordpress-mobile/AztecEditor-Android#417]
Links should be underlined for improved accessibility.

**Comment by iamthomasbishop** [comment]
Side Note: Links should also be set in Blue Medium, same as Calypso. https://wordpress.com/design-handbook/colors/

**Comment by maxme** [comment]
Links are underlined and they are blue in wpandroid.
